sdk/batch/Microsoft.Azure.Batch/tests/IntegrationTests/AddTaskCollectionIntegrationTests.cs

```diff
@@ -467,7 +467,7 @@ public async Task Bug1360227_AddTasksBatchTimeout(bool useJobOperations)
                             batchCli,
                             testName,
                             311,
-                            timeout: TimeSpan.FromSeconds(1),
+                            timeout: TimeSpan.FromSeconds(-1),
```

Comment:
Seems like we should settle on a timeout somewhere between 1 second and infinite

Reply:
This fixes a flaky test that is supposed to timeout.

Reply:
The timespan is eventually added to the start time of the task, to determine an absolute timeout time: https://github.com/Azure/azure-sdk-for-net/blob/4fe2be5d7155f897576d9746692e33dd20360c9b/sdk/batch/Microsoft.Azure.Batch/src/AddTasksWorkflowManager.cs#L103

Reply:
Thanks, that makes sense - I misinterpreted the -1 as an infinite timeout.